README.md

```diff
@@ -63,6 +63,9 @@
 
 <!-- GETTING STARTED -->
 
+## 🛠 Live Demo <a name="live-demo"></a>
+Check it out [here]( https://estete9.github.io/money-history/)
+
 ## 💻 Getting Started <a name="getting-started"></a>
 
```

Comment:
When I try and open your live demo link its throwing an error please kindly fix this issue.

Kindly deploy your project using render or netlify



Secondly you did not record a video presentation showcasing your project:
Video presentation

Provide feedback about the video presentation in the code review summary.

The student recorded a video presenting the features of the project.
The maximum length of the video is 5 minutes.
The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
The video contains a demo of the project, highlighting all the project features.
The video contains a highlight about a piece of the code the student finds interesting or is proud of making.


Note: It's recommended that students use [Loom](https://www.loom.com/) to record to video but they can use any tool they want as long as they share an online link of the uploaded video.

It would be nice if you recorded a video based on the above requirements 

Third you did not Acknowledge the author of the original design

Please follow these requirements:
Creative Commons attribution


The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).